pkg/api/azenvtypes.go

```diff
@@ -221,6 +230,7 @@ var (
 			AKSUbuntu1804:     AKSUbuntu1804OSImageConfig,
 			AKS1804Deprecated: AKSUbuntu1804OSImageConfig, // for back-compat
 			ACC1604:           ACC1604OSImageConfig,
+			AKSGPU1804:        AKSGPU1804OSImageConfig,
```

Comment:
We should also update
AzureUSGovernmentCloudSpec 
AzureChinaCloudSpec

Reply:
added. btw I don't see ACC1604 in USGov and China Cloud, are they not enabled?

Reply:
we don't plan to support 1604 for gen2, since we want to move to 1804